src/plugins/flow.js

```diff
@@ -1223,6 +1223,10 @@ export default function (instance) {
         specifier.local = specifier.imported.__clone();
       }
 
+      if (node.importKind !== "value" && specifier.importKind !== null) {
```

Comment:
Per this discussion with @loganfsmyth, I guess this should stick to explicitly listing out `"type"` and `"typeof"`

https://github.com/babel/babel/pull/5035#discussion_r93822476